client/src/pages/Experience/Experience.js

```diff
@@ -2,26 +2,39 @@ import React, { useState, useEffect } from "react";
 import SignUpContainer from "../../components/SignUpContainer";
 import NewExperienceForm from "./NewExperienceForm";
 import AddExperienceButton from "./AddExperienceButton";
+import { availableLanguages } from "../../utils";
 
-// TO-DO: ADD FUNCTIONALITY TO FILTER LANGUAGES ALREADY SELECTED
-const Experience = props => {
+const Experience = () => {
   const [experience, setExperience] = useState([{ C: 1 }]);
```

Comment:
Lol that's a funny default

Reply:
I feel like a better and more efficient structure of experience is as an object, like this:
{
"C": 1
}
It'll be easier to update